src/api/app.py

```diff
@@ -7,16 +7,22 @@
 
 import requests
 import sqlalchemy as sa
-from litestar import Litestar, MediaType, Request, get, websocket, websocket_listener
+from litestar import Litestar, MediaType, Request, WebSocket, get, post, websocket_listener
 from litestar.connection import ASGIConnection
 from litestar.datastructures import State
 from litestar.di import Provide
 from litestar.exceptions import NotAuthorizedException
+from litestar.handlers import WebsocketListener
 from litestar.handlers.base import BaseRouteHandler
 from litestar.response import Redirect
 from sqlalchemy import Engine, create_engine
 from sqlalchemy.ext.asyncio import AsyncEngine, create_async_engine
 
+from api.lib import generate_uuid4_int
+
+DEMOS_PATH = os.path.expanduser(os.path.join("~/media", "demos"))
+os.makedirs(DEMOS_PATH, exist_ok=True)
+
 
 def _make_db_uri(async_url: bool = False) -> str:
```

Comment:
Duplicate of the function added to lib. Currently it seems this is the one being called and used since the new function is not imported.